presto-main/src/main/java/com/facebook/presto/server/HttpRemoteTask.java

```diff
@@ -482,6 +513,7 @@ public synchronized void abort()
         try (SetThreadName ignored = new SetThreadName("HttpRemoteTask-%s", taskId)) {
             // clear pending splits to free memory
             pendingSplits.clear();
+            fireSplitChanged(pendingSourceSplitCount);
```

Comment:
wrap this four lines related to clearing pendingSplits as method as already it is suggested by a comment